src/frontend/next/src/contexts/themeContext.ts

```diff
@@ -0,0 +1,8 @@
+import { createContext } from 'react';
+
+const ThemeContext = createContext({
+  themeType: 'light',
+  toggleTheme: () => {},
```

Comment:
Why not provide a way to switch?  What is the point of this?

Reply:
@humphd : the `ThemeToggleButton` uses `themeType` for conditional rendering the icon and use the `toggleTheme` to switch the value. I just don't want to pass down to 3 levels of components from App -> Header -> DesktopHeader/MobileHeader -> ThemeToggleButton. So I create a context to simply use it in the` 
ThemeToggleButton`

Reply:
I understand the use of the context, I mean something different.  Above you have code like this:

```jsx
<IconButton onClick={toggleTheme} className={classes.themeToggleButton}>
```

It's using the `toggleTheme` function you define here, but that function is doing nothing.  Why does it not literally toggle the theme?

Reply:
I tried to do move toggleTheme function from _app to here but found some difficulty to rerender the app.